### PR TITLE
Add optional save flag to luis:application:import

### PR DESCRIPTION
**Pull request by jspruance**

### CODE REVIEW

**Comment by jspruance** [comment]
Customers would like to use 'luis:application:import' to import a new app based on a json file, but it is difficult to programmatically retrieve the newly created app Id afterwards.

Proposed change: add an optional 'save' flag so that the appId can be saved to config and be readily available for use for downstream operations. This is the exact same functionality currently available in 'luis:application:create'.

File: packages/luis/README.md [hunk 1]
```diff
@@ -43,6 +45,534 @@ OPTIONS
   -h, --help  LUIS command help
 ```
```
Comment (jspruance) [inline packages/luis/README.md]: hmm...the url correction script didn't seem to run here...looking into it

Comment (jspruance) [reply packages/luis/README.md]: fixed

File: packages/luis/README.md [hunk 2]
```diff
@@ -9,6 +9,8 @@
 <!-- toc -->
 * [Commands](#commands)
 <!-- tocstop -->
+* [Commands](#commands)
```
Comment (munozemilio) [inline packages/luis/README.md]: Fix duplicated tags

File: packages/luis/src/commands/luis/application/import.ts [hunk 3]
```diff
@@ -47,9 +48,35 @@ export default class LuisApplicationImport extends Command {
...
       const response = await client.apps.importMethod(JSON.parse(appJSON), options)
       this.log(`App successfully imported with id ${response.body}.`)
+      if (save) {
+        const config = {
+          appId: response.body,
+          endpoint,
+          subscriptionKey
+        }
+        const saveConfigResponse = await this.saveImportedConfig(config, configDir)
+        if (saveConfigResponse) this.log('Config settings saved')
```
Comment (munozemilio) [inline packages/luis/src/commands/luis/application/import.ts]: this is always true

Comment (jspruance) [reply packages/luis/src/commands/luis/application/import.ts]: fixed

File: packages/luis/README.md (hunk 2 above)
Comment (jspruance) [reply packages/luis/README.md]: fixed

File: packages/cli/README.md [hunk 4]
```diff
@@ -15,6 +15,1585 @@ $ npm install -g @microsoft/botframework-cli
 
 # Commands
 <!-- commands -->
+* [`bf `](#bf-)
```
Comment (munozemilio) [inline packages/cli/README.md]: is this duplicated too?

File: packages/luis/src/commands/luis/application/import.ts [hunk 5]
```diff
@@ -47,9 +48,35 @@ export default class LuisApplicationImport extends Command {
     try {
       const response = await client.apps.importMethod(JSON.parse(appJSON), options)
       this.log(`App successfully imported with id ${response.body}.`)
+      if (save) {
+        const config = {
+          appId: response.body,
+          endpoint,
+          subscriptionKey
+        }
+        const saveConfigResponse = await this.saveImportedConfig(config, configDir)
```
Comment (munozemilio) [inline packages/luis/src/commands/luis/application/import.ts]: Is this const used?

File: packages/luis/README.md [hunk 6]
```diff
@@ -49,6 +49,534 @@ _See code: [src/commands/luis/index.ts](https://github.com/microsoft/botframewor
 
 Creates a new LUIS application
 
+```
```
Comment (munozemilio) [inline packages/luis/README.md]: Ideally the change should be only in the luis:application:import. this looks a bit odd, is this correct?

**Comment by cleemullins** [comment]
@munozemilio Please resolve this today, as @jspruance turns into a pumpkin today at 5pm.

**Pull request closed without merging**